requirements/REQUIREMENTS.md

```diff
@@ -0,0 +1,27 @@
+##Предполагаемая ЦА
+Пользователи мессенджеров, соцсетей и иных сервисов, которых не удовлетворяет функционал данных сервисов в области опросов.
+
+##Разбиение ЦА
++ Соцсети
++ Мессенджеры
++ Пользователи-компании (использование во внутренних приложениях организаций)
+
+##Use cases для каждой группы ЦА
+![use-cases](../img/use-cases.png)
```

Comment:
What is going on the diagram? Have you taken a look at the examples of use case diagrams?